laravel-app/app/Interfaces/PostServiceInterface.php

```diff
@@ -8,8 +8,14 @@
 interface PostServiceInterface
 {
     /**
-     * @param User $user
      * @param array<mixed, mixed> $validatedData
      */
     public function createPost(User $user, array $validatedData): UserPost;
+
+    /**
+     * @param array<string, mixed> $validatedData
```

Comment:
removeee!!

Reply:
Apologies sir, but if I remove the @param for the arrray the larastan checker throws error and requires that to be added. 🙇